libraries/AP_Follow/AP_Follow.cpp

```diff
@@ -422,6 +431,10 @@ bool AP_Follow::get_target_location_and_velocity(Location &loc, Vector3f &vel_ne
     }
     vel_ned = _estimate_vel_ned_ms;
 
+    // The frame requested by FOLL_ALT_TYPE may not be the frame of location returned by ahrs. 
+    // Make sure we give the caller the frame they have asked for.
+    loc.change_alt_frame(_alt_type);
```

Comment:
Do all the users of this method deal with all alt frame types correctly? `Log_Write_FOLL` is assuming abs.